src/Schema.js

```diff
@@ -802,17 +804,23 @@ const makeDescriptor = function (prop, schema, opts) {
     // TODO: Make it so tracking can be turned on for all properties instead of
     // only per-property
     if (track && !_get(creatingPath)) {
+      /* previous is versioned on database commit
+       * props are versioned on set()
+       * TODO? Option to disable props versioning (commit only versioning)
+       */
       const previous = _get(previousPath)
       const current = _get(keyPath)
       let changing = _get(changingPath)
       let changed = _get(changedPath)
 
+      // IS it possible for changing to be defined but changed undefined?
```

Comment:
It's possible, someone might be messing with these "private" properties during an event loop. 😕  If this is the first time in the current event loop that a property has changed, then need to initialize `changed` to an empty array.